greta-dev/greta
# v0.3.0 release candidate

to do before merging:
 - [x] improve tensorflow installation instructions for Anaconda
 - [x] test against latest tensorflow
 - [x] pass `R CMD check --as-cran` on required systems
 - [x] pass `devtools::release()` checks
 - [x] pass local `lintr::lint_package()` checks
 - [x] be accepted on CRAN


after merging:
 - create a github release
 - add to zenodo
